lib/WWW/WebKit2.pm

```diff
@@ -361,8 +361,7 @@ sub init_webkit {
             my $action = $decision->get_navigation_action;
             my $action_uri = $action->get_request->get_uri;
 
-            unless ($action_uri eq 'about:blank') {
-
+            unless ($action_uri =~ m/about:\w+$/) {
```

Comment:
This would also find `http://test.example/about:me`
`m/\A about:/x` seems more appropriate

Reply:
True. Has been changed.